packages/forma-36-tokens/CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### Features
 
-* new colors tokens update ([#1041](https://github.com/contentful/forma-36/issues/1041)) ([fc0beba](https://github.com/contentful/forma-36/commit/fc0beba32310d2ab31e59006f6701f6ab7c0f79f))
+* Updated color system that contains new set of tokens with new namespaces and codemod to improve the experience of the migration ([#1041](https://github.com/contentful/forma-36/issues/1041)) ([fc0beba](https://github.com/contentful/forma-36/commit/fc0beba32310d2ab31e59006f6701f6ab7c0f79f))
```

Comment:
```suggestion
* Updated color system that contains a new set of tokens with new namespaces ([#1041](https://github.com/contentful/forma-36/issues/1041)) ([fc0beba](https://github.com/contentful/forma-36/commit/fc0beba32310d2ab31e59006f6701f6ab7c0f79f))
```

I don't think the codemod is a part of the tokens so I'd say we should mention it separately but still keep it easy to find.

Reply:
yeah, it was part of the same PR, thats' why I thought I would add it here. Do you suggest a separate mention of in below? How it should look like in the changelog?

Reply:
yeah, sorry, codemod has its own changelog.